modules/sdk/src/main/scala/org/tessellation/sdk/app/TessellationIOApp.scala

```diff
@@ -137,7 +139,7 @@ abstract class TessellationIOApp[A <: CliMethod](
                               session,
                               p2pClient.node,
                               validators,
-                              _seedlist,
+                              _seedlist |+| _prioritySeedlist,
```

Comment:
We need to discuss this the line below where we're merging the seedlists.
Things that I'm not sure about:

1. How early/where can we merge these w/o affecting correctness (e.g. validation? registration?)
2. Is this a strong case for @marcinwadon 's suggestion for using a single seedlist but with an added priority flag?

Reply:
I belive we should never merged them 🤔 as @ryle-goehausen said, the priority seedlist should be a subset of the original seedlist

Reply:
@TheMMaciek when "priority seedlist should be a subset of the original seedlist" it means you would never have additional nodes on the priority seedlist that you don't have on the seedlist. also, you may add fields to the original seedlist as the check during joining is against `seedlist.map(_.peerId)` so we check only the list of peers, not the whole seedlist

Reply:
@marcinwadon So you're saying the priority seedlist will always be a subset of original seedlist? @ryle-goehausen  can you confirm?

@TheMMaciek The point of merging is so that the priority peers will be joined and stored in the cluster storage, otherwise the majority picking algorithm won't see them.

Reply:
Also keep in mind that `_seedlist` here represents a seedlist for this particular network and not a GL0 network seedlist as I mentioned in another comment and to some extent above as well.